scripts/data-statistics.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+const bcd = require('..');
+
+const browsers = ['chrome', 'chrome_android', 'edge', 'firefox', 'ie', 'safari', 'safari_ios', 'webview_android'];
+let stats = { total: { all: 0, true: 0, null: 0, real: 0 } };
+browsers.forEach(browser => {
+  stats[browser] = { all: 0, true: 0, null: 0, real: 0 }
+});
+
+const processData = (data) => {
+  if (data.support) {
+    browsers.forEach(function(browser) {
+      stats[browser].all++;
+      stats.total.all++;
+      let real_value = true;
+      if (!data.support[browser]) {
+        stats[browser].null++;
+        stats.total.null++;
+        real_value = false;
+      } else {
+        if (JSON.stringify(data.support[browser]).includes('null')) {
+          stats[browser].null++;
+          stats.total.null++;
+          real_value = false;
+        }
+        if (JSON.stringify(data.support[browser]).includes('true')) {
+          stats[browser].true++;
+          stats.total.true++;
+          real_value = false;
+        }
+      }
+      if (real_value) {
+        stats[browser].real++;
+        stats.total.real++;
+      }
+    });
+  }
+};
+
+const iterateData = (data) => {
+  for (let key in data) {
+    if (key == '__compat') {
+      processData(data[key]);
+    } else {
+      iterateData(data[key]);
+    }
+  }
+};
+
+for (let data in bcd) {
+  if (!(data === 'browsers' || data === 'webextensions')) {
+    iterateData(bcd[data]);
+  }
+}
+
+const printTable = () => {
+  let table = `| browser | real values | true values | null values |
+| --- | --- | --- | --- |
+`;
+
+  Object.keys(stats).forEach(entry => {
+    table += `| ${entry.replace('_', ' ')} | `;
+    table += `${((stats[entry].real / stats[entry].all) * 100).toFixed(2)}% | `;
+    table += `${((stats[entry].true / stats[entry].all) * 100).toFixed(2)}% | `;
+    table += `${((stats[entry].null / stats[entry].all) * 100).toFixed(2)}% |
+`;
+  });
+
+  console.log(table);
+}
+
+console.log('Status as of version 0.0.xx (released on xx/xx/2019) for web platform features: \n')
```

Comment:
This seems fine for now, but at some point I'd like to fold this into the release script so you don't have to fill in the blanks here

Reply:
I'm unsure how to fold this into the release script exactly. Should it just print it at the end?
Maybe I can leave this for you to follow-up on?

Reply:
Yeah, leave this for now—I can pick this up once it's merged. I think the script deserves to standalone even if it is part of the release notes